dts/arm/st/f4/stm32f429Xi.dtsi

```diff
@@ -6,6 +6,7 @@
 
 #include <mem.h>
 #include <st/f4/stm32f429.dtsi>
+#include <dt-bindings/memory-controller/stm32-fmc-sdram.h>
```

Comment:
Please move this one to dts/arm/st/f4/stm32f405.dtsi as well (/stm32f4.dtsi) alternatively.

Reply:
I guess it should be moved to every file defining fmc/sdram node.

Reply:
Actually a few more are supported: https://www.st.com/en/microcontrollers-microprocessors/stm32f4-series.html